app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -388,7 +388,14 @@ public void showBadImagePopup(Integer errorCode,
                 getString(R.string.cancel),
                 () -> {
                     uploadItem.setImageQuality(ImageUtils.IMAGE_KEEP);
-                    onNextButtonClicked();
+                    /*
+                        This "onNextButtonClicked();" function creating the issue #4806
+
+                        user skipped the warning of low quality image but, onNextButtonClicked
+                        function again checking the quality of image, that's why replacing it with
+                        below function solves the issue.
+                    */
```

Comment:
In this project we usually write javadoc to explain the current code, rather than past code. Could you please change to this?
`User skipped the warning of low quality image, so we call onImageValidationSuccess rather than onNextButtonClicked to avoid showing other warning popups again.`

Reply:
Olay, i will change it

Reply:
*okay